src/index.js

```diff
@@ -1,17 +1,38 @@
-import React from 'react';
-import ReactDOM from 'react-dom';
-import './index.css';
-import App from './App';
-import reportWebVitals from './reportWebVitals';
+import arrayShuffle from "array-shuffle"
+import React from "react"
+import ReactDOM from "react-dom"
+import { BrowserRouter, Route, Switch } from "react-router-dom"
+import ContributorDetails from "./ContributorDetails"
+import entries from "./entries"
 
-ReactDOM.render(
-  <React.StrictMode>
-    <App />
-  </React.StrictMode>,
-  document.getElementById('root')
-);
+const generateRoutes = (entries) => {
+    if(entries.length === 0){
+        return null
+    } 
```

Comment:
```js
if (!entries.length) return null
```

Reply:
better if include the typing check with ===. since this is not typed.